tools/releaseBuild/azureDevOps/templates/release-MsixBundle.yml

```diff
@@ -0,0 +1,45 @@
+parameters:
+  jobName: ""
+  displayName: ""
+  imageName: ""
+  globalToolExeName: 'pwsh.exe'
+  globalToolPackageName: 'PowerShell.Windows.x64'
+
+
+jobs:
+- job: ${{ parameters.jobName }}
+  displayName: ${{ parameters.displayName }}
+  pool:
+    vmImage: ${{ parameters.imageName }}
+  steps:
+    - checkout: self
+      clean: true
+
+    - task: DownloadPipelineArtifact@2
+      inputs:
+        source: specific
+        project: PowerShellCore
+        pipeline: '696'
+        preferTriggeringPipeline: true
+        runVersion: latestFromBranch
+        runBranch: '$(Build.SourceBranch)'
+        artifact: finalResults
+        patterns: '**/*.msix'
+        path: '$(Pipeline.Workspace)/msix'
+
+    - pwsh: |
+        if ($null -eq Get-Command makeappx -ErrorAction Ignore) {
+          $msixUrl = '$(makeappUrl)'
```

Comment:
Where is `makeappUrl` defined?

Reply:
It's a library variable in MSCodeHub

Reply:
You should define that you are using it.

Reply:
Here is an example:  https://github.com/PowerShell/PowerShell/blob/e475e5382d787fe4ab84ebb0faf816c4eecc11cc/tools/releaseBuild/azureDevOps/templates/windows-package-signing.yml#L15